src/modules/test/recipes/CommandRunnerTests.json

```diff
@@ -1,40 +1,124 @@
 [
-    {
-        "ComponentName": "CommandRunner",
-        "ObjectName": "commandStatus",
-        "Desired": false,
-        "ExpectedResult": 0,
-        "WaitSeconds": 0
-    },
-    {
-        "ComponentName": "CommandRunner",
-        "ObjectName": "commandArguments",
-        "Desired": true,
-        "Payload": "{\"commandId\": \"helloworld\", \"arguments\": \"echo Hello World!\", \"timeout\": 0, \"singleLineTextResult\": true, \"action\": 100, }",
-        "ExpectedResult": 22,
-        "WaitSeconds": 0
-    },
-    {
-        "ComponentName": "CommandRunner",
-        "ObjectName": "commandArguments",
-        "Desired": true,
-        "Payload": "{\"commandId\": \"test1\", \"arguments\": \"echo Hello World\", \"timeout\": 0, \"singleLineTextResult\": true, \"action\": 3}",
-        "ExpectedResult": 0,
-        "WaitSeconds": 5
-    },
-    {
-        "ComponentName": "CommandRunner",
-        "ObjectName": "commandStatus",
-        "Desired": false,
-        "Payload": "{\"commandId\":\"test1\",\"resultCode\":0,\"textResult\":\"Hello World \",\"currentState\":2}",
-        "ExpectedResult": 0
-    },
-    {
-        "ComponentName": "CommandRunner",
-        "ObjectName": "commandArguments",
-        "Desired": true,
-        "Payload": "{\"commandId\": \"removeCache\", \"arguments\": \"rm /etc/osconfig/osconfig_commandrunner.cache\", \"timeout\": 0, \"singleLineTextResult\": true, \"action\": 3}",
-        "ExpectedResult": 0,
-        "WaitSeconds": 1
-    }
+  {
+    "ComponentName": "CommandRunner",
+    "ObjectName": "commandStatus",
+    "Desired": false,
+    "ExpectedResult": 0,
+    "WaitSeconds": 0
+  },
+  {
+    "ComponentName": "CommandRunner",
+    "ObjectName": "commandArguments",
+    "Desired": true,
+    "Payload": "{\"commandId\":\"TestCommandOne\",\"arguments\":\"echo Test\",\"timeout\":0,\"singleLineTextResult\":true,\"action\":3}",
+    "PayloadSizeBytes": 0,
```

Comment:
not necessary, default=0 which, uses the size of the payload defined above

Reply:
Good! I will remove then